src/api/public/apidocs-new/OBS-v2.10.50.yaml

```
@@ -18,6 +18,30 @@ tags:
   - name: Published Binaries
   - name: Requests
   - name: Search
+    description: |
+      All endpoints besides the owner search listed in the search group use Xpath expressions. The `match` query parameter is used
+      to pass the Xpath predicate. The individual endpoints refer to different objects like Packages, Projects, Issues and more...
+      Please refer to the documentation of the individual search endpoints to know which attributes can be accessed through the predicate,
+      and which extra query parameters are supported or might be required in order to run the search.
+
+
+      These functions can be used with the predicates: `boolean`, `contains`, `not`, `starts_with`, `ends_with`.
+
+      _Example_:
+
+      For a list of all packages with the name "obs-server" that are part of projects where the name starts with "OBS:".
+
+      `/search/package?match=@name='obs-server'+and+starts_with(@project,'OBS:')`
```

```suggestion
      For a list of all packages that are part of projects where the name starts with "OBS:".

      `/search/package?match=starts_with(@project,'OBS:')`
```
I suggest to show in this example only the use of a function, without the `and` operator. It's more clear to introduce concepts once at a time.

---

makes sense, done